libs/ui-watt/src/lib/components/button/watt-button-alt.component.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+import { WattIcon } from '../../foundations/icon';
+
+export type WattButtonVariant = 'primary' | 'secondary' | 'text' | 'icon';
+export type WattButtonSize = 'normal' | 'large';
+
+@Component({
+  selector: 'watt-button-alt',
+  template: `
+    <button mat-button [ngClass]="[variant, size]" [disabled]="disabled">
```

Comment:
issue: quick observation (haven't done a complete review) - adding classes like `primary`, `secondary`, `normal`, etc. might end up being targeted by unrelated CSS because they are too general. Consider renaming the classes to `watt-button--primary`, `watt-button--secondary`, etc.

Reply:
Agreed, as long the style encapsulation is none we should prefix our classes

Reply:
Fixed :)

Reply:
Nice work so far! My previous comment should've been more specific but the `size` class has the same issues - `normal` and `large` are too general. Maybe `watt-button--normal` and `watt-button--large`?

Reply:
Oh right :) My bad! Will get it up now :)

Reply:
Done @dzhavat 😄